CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# v6.0.0(unreleased)
```

Comment:
Since this new release will just be adding a `new resource/module`, and not breaking backwards compatibility, according to the [Semantic Versioning](http://semver.org/) this should actually be `v5.1.0`